Description: Use Debian's imaplib2 module instead of the bundled version.
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
Forwarded: https://github.com/AyatanaIndicators/ayatana-webmail/issues/14

--- a/ayatanawebmail/application.py
+++ b/ayatanawebmail/application.py
@@ -22,7 +22,7 @@
 import sys
 import time
 import re
-import ayatanawebmail.imaplib2 as imaplib
+import imaplib2 as imaplib
 import os.path
 import os
 import urllib3
--- a/ayatanawebmail/dialog.py
+++ b/ayatanawebmail/dialog.py
@@ -5,7 +5,7 @@
 #          Robert Tari <robert@tari.in>
 # License: GNU GPL 3 or higher; http://www.gnu.org/licenses/gpl.html
 
-import ayatanawebmail.imaplib2 as imaplib
+import imaplib2 as imaplib
 import os.path
 from socket import error as socketerror
 from gi.repository import Gtk, GdkPixbuf, Gdk
